src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println(this.getClass().getSimpleName() + " started work");
```

Comment:
```suggestion
        System.out.println("Bulldozer started work");
```

The purpose of this task is to explore the principles of OOP, let's explicitly write down the name of the objects, so that the implementation of the methods would differ in child classes